libbeat/kibana/index_pattern_generator_test.go

```diff
@@ -37,7 +37,7 @@ func TestNewGenerator(t *testing.T) {
 
 	v, _ := common.NewVersion("7.0.0")
 	// checks for fields.yml
-	generator, err := NewGenerator("beat-index", "mybeat.", filepath.Join(beatDir, "notexistent"), "7.0", *v)
+	generator, err := NewGenerator("beat-index", "mybeat.", filepath.Join(beatDir, "nonexistent"), "7.0", *v)
```

Comment:
hopefully not a problem

Reply:
I assume it will break the tests. We should also rename the directory.